src/Microsoft.SqlTools.Migration/MigrationService.cs

```diff
@@ -580,6 +583,62 @@ public void InitializeService(IProtocolEndpoint serviceHost)
             }
         }
 
+        internal async Task HandleGetArmTemplateRequest(
+    string targetType,
+    RequestContext<string> requestContext)
+        {
+            ProvisioningScriptServiceProvider provider = new ProvisioningScriptServiceProvider();
```

Comment:
Suggestion: using try catch for this function.

Reply:
done